src/config/SettingsDataEntries.tsx

```diff
@@ -189,16 +189,13 @@ export const ConfigurableMessageKeys = (
 	{
 		name: 'Join DMs',
 		key: 'messagesJoinDM',
-		placeholder: [guildSettings.eventsMemberAdd ? null : 'You must configure Member Join on the Events page.'].join(' '),
+		placeholder: '',
```

Comment:
```suggestion
```
Make placeholder optional in `ConfigurableData.d.ts` line 58. The place where it gets used as property to a React component also accepts undefined.